corehq/apps/app_manager/management/commands/migrate_apps.py

```diff
@@ -0,0 +1,61 @@
+from corehq.toggles import SYNC_SEARCH_CASE_CLAIM
+from django.core.management.base import BaseCommand
+from corehq.apps.app_manager.models import Application
+from corehq.apps.app_manager.management.commands.helpers import get_all_app_ids
+from couchdbkit.exceptions import ResourceNotFound
+from lxml.etree import XMLSyntaxError
+
+
+class Command(BaseCommand):
```

Comment:
If you extend `AppMigrationCommandBase` you can just override a few methods and the base class will handle the rest for you:

```python
class Command(AppMigrationCommandBase):
    include_builds = False  # not sure if this should be True of False
    include_linked_apps = False  # not sure if this should be True of False
    DOMAIN_LIST_FILENAME = "case_search_title_migration_domains"
    DOMAIN_PROGRESS_NUMBER_FILENAME = "case_search_title_migration_progress"

    def get_domains(self):
        return list(sorted(SYNC_SEARCH_CASE_CLAIM.get_enabled_domains()))

    def migrate_app(self, app):
        # your implementation